CMakeLists.txt

```diff
@@ -6,12 +6,16 @@ endif()
 
 cmake_minimum_required(VERSION ${REQUIRED_CMAKE_VERSION})
 
-option(BUILD_SHARED_LIBS "Build external projects as shared libraries." ON)
+if(APPLE)
```

Comment:
The previous version did not work correctly.

"BUILD_SHARED_LIBS" is a Cache variable, which means "set(BUILD_SHARED_LIBS OFF)" on line 56 fails to change the value to OFF. There were two ways to fix this: FORCE the change on line 56, or change it when first defined. I thought it would be clearer to set the value up front, rather than having a change in value buried in the code.